modules/olm-installing-po-after.adoc

```diff
@@ -0,0 +1,93 @@
+// Module included in the following assemblies:
+//
+// * operators/admin/olm-managing-po.adoc
+
+:_content-type: PROCEDURE
+[id="olm-installing-po-after_{context}"]
+= Installing platform Operators after cluster creation
+
+As a cluster administrator, you can install platform Operators after cluster creation on clusters that have enabled the `TechPreviewNoUpgrades` feature set by using the cluster-wide `PlatformOperator` API.
+
+.Prerequisites
+
+- Access to an {product-title} cluster using an account with `cluster-admin` permissions.
+- Enable the `TechPreviewNoUpgrades` feature set on the cluster. See _Additional resources_ for details.
+- Install the `oc` command to your local system.
+
+.Procedure
+
+. Choose a platform Operator from the supported set of OLM-based Operators. See _Additional resources_ for the list of this set and details on current limitations.
+
+. Create an `PlatformOperator` object YAML file for your chosen platform Operator, for example a `cert-manager.yaml` file for the {cert-manager-operator}:
++
+[source,yaml]
+----
+apiVersion: platform.openshift.io/v1alpha1
+kind: PlatformOperator
+metadata:
+  name: cert-manager
+spec:
+  package:
+    name: openshift-cert-manager-operator
+----
+
+. Create the `PlatformOperator` object:
++
+[source,terminal]
+----
+$ oc apply -f cert-manager.yaml
+----
+
+. Wait until the `cert-manager` platform Operator has reported an `Installed=true` status condition:
++
+[source,terminal]
+----
+$ oc get platformoperator cert-manager -o yaml
+----
++
+.Example output
+[source,yaml]
+----
+...
+status:
+  activeBundleDeployment:
+    name: cert-manager
+  conditions:
+  - lastTransitionTime: "2022-10-24T17:24:40Z"
+    message: Successfully applied the cert-manager BundleDeployment resource
+    reason: InstallSuccessful
+    status: "True"
+    type: Installed
```

Comment:
Then you could add callouts to the parts they are looking for explaining what success looks like, or possibly just add it as a result line under this block.

Just a thought!